src/connectors/magnatune.js

```diff
@@ -0,0 +1,35 @@
+
+
+'use strict';
+
+const artistSelector = "a[href^='/artists']";
+
+const trackRegex = /(.+)\[(\.)?/;
+
+Connector.playerSelector = '#paper';
+
+Connector.artistSelector = artistSelector;
+
+Connector.getTrack = () => {
+	const playingImg = document.querySelector('#songlist img');
+
+	if (!playingImg) {
+		return null;
+	}
+
+	const trackContent = playingImg.closest('td').nextElementSibling.textContent;
+	const matched = trackContent.match(trackRegex);
+	if (!matched) {
+		return null;
+	}
+
+	return matched[1].trim();
+};
+
+Connector.getAlbum = () => {
+	const artistElement = document.querySelector(artistSelector);
+
+	return artistElement.closest('font').innerText.replace(artistElement.textContent, '').slice(1);
```

Comment:
`innerText` can be replaced with `textContent`.

Reply:
This always gets me!